samples/msal-node-samples/custom-INetworkModule-and-network-tracing/HttpClientCurrent.ts

```diff
@@ -287,7 +296,7 @@ const networkRequestViaHttps = <T>(
     }
 
     return new Promise<NetworkResponse<T>>((resolve, reject) => {
-        const request = https.request(url, customOptions);
```

Comment:
Does the bug still happen if you use `https.request(url: Url, customOptions: Options)` instead of `https.request(url: string, customOptions: Options)` ?

Reply:
Yep